modules/standard/FileSystem.chpl

```diff
@@ -427,6 +427,29 @@ proc sameFile(file1: file, file2: file): bool {
   return result;
 }
 
+/* Create a symbolic link pointing to actual named sym.  May generate an error
+   message.
+   err: a syserr used to indicate if an error occurred during creation
+   actual: the source file to be linked
+   symLink: the location the symbolic link should live
```

Comment:
Because I'm lazy and just copy+pasted this from my proposal, ha ha.  I am intending to update it, but all in one go.